web_src/less/_repository.less

```diff
@@ -3068,6 +3068,43 @@ td.blob-excerpt {
   padding-left: 8px;
 }
 
+#diff-container {
+  display: flex;
+}
+#diff-file-boxes {
+  width: 100%;
+}
+
+#diff-file-tree {
+  width: 20%;
+  max-width: 380px;
+  line-height: inherit;
+  position: sticky;
+  padding-top: 0;
+  top: 47px;
+  max-height: calc(100vh - 50px);
+  height: calc(100vh - 50px);
+  overflow-y: auto;
+}
+
+#diff-file-tree[style*='display: block'] ~ #diff-file-boxes {
```

Comment:
Seems fragile when using `[style*='display: block']`, could there a dedicate CSS class for the hidden tree?

IIRC, Gitea's CSS (Fomantic UI) has a `hidden` attribute, which works like `<div hidden>` if there is no other `display` style. If there is already a `display` attribute on the element, then it needs a new CSS class.

Reply:
Hmm, flex works better. I made some changes on this PR directly.